mindone/trainers/callback.py

```diff
@@ -267,7 +267,9 @@ def on_train_epoch_end(self, run_context):
                     self.ema.swap_before_eval()
 
                 # save history checkpoints
-                self.ckpt_manager.save(self.net_to_save, None, ckpt_name=ckpt_name, append_dict=append_dict)
+                self.ckpt_manager.save(
+                    self.net_to_save, perf=cb_params["net_outputs"], ckpt_name=ckpt_name, append_dict=append_dict
```

Comment:
why add perf=cb_params["net_outputs"]? does it fit for other models?

Reply:
For all current examples, eval callback is setup as `latest_k`. However, I found that this model's latest trained ckpt is not necessarily good. Thus, I passed in `top_k` to the callback, which requires loss to be recorded under this line. This modification does not affect `latest_k` setup in other examples.